test/functional/api/mark_extended_spec.lua

```diff
@@ -0,0 +1,743 @@
+-- TODO(timeyyy):
+-- do_bang needs to be tested
+-- do_sub needs to be tested
+-- diff needs to be tested
+-- do_filter needs to be tested
+-- filter_lines needs to be tested (mark_col_adjust)
```

Comment:
Are these todos still relevant?

Reply:
It would be nice I suppose,  I never manually or otherwise tested tested that extmarks move with those functions. I was not sure how to hit the code paths